{"uuid": "afab54d3-ff56-4b64-9148-e9046f0a57c4", "children": ["656162cc-4fd3-49ae-881a-67fb14fb4bf8", "1c64f596-c3c9-4653-a094-efa3fcdfa713", "b6397fe6-87d3-438c-a12b-cfce8be33e0b", "1247d7bd-b1f7-4c05-aee8-765e949e39d1"], "befores": [{"name": "start_class_App", "status": "passed", "start": 1649260139285, "stop": 1649260153627}], "afters": [{"name": "start_class_App::0", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started\n\n", "trace": "  File \"d:\\python\\python38\\lib\\site-packages\\allure_commons\\_allure.py\", line 200, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"d:\\python\\python38\\lib\\site-packages\\_pytest\\fixtures.py\", line 933, in _teardown_yield_fixture\n    next(it)\n  File \"D:\\JetBrains\\PyCharm\\PycharmProjects\\Duixi_Framework_V1\\TestCases\\Test0Welcome\\conftest.py\", line 56, in start_class_App\n    driver.quit()\n  File \"d:\\python\\python38\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 698, in quit\n    self.execute(Command.QUIT)\n  File \"d:\\python\\python38\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"d:\\python\\python38\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 31, in check_response\n    raise wde\n  File \"d:\\python\\python38\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 26, in check_response\n    super().check_response(response)\n  File \"d:\\python\\python38\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1649260242299, "stop": 1649260242313}], "start": 1649260139285, "stop": 1649260242313}